### ISSUE TITLE
Minor nuisance with `...` expansion and `foreach`

### ISSUE THREAD

**Issue by Rmano** [lervag/vimtex#301]
The expansion from `...` to `\dots` is nice and useful, 99% of the time. 
I have just one problem and I like to ask if there is a nicer solution than the one I use... 
When using `tikz`, I often write: 

```
foreach \x in {1,2,...,9} 
```
...and to write itI have to type two dots, wait a bit, and then continue. Is there a better way? For example adding a space to `...` expansion would break things?

**Comment by lervag** [comment]
Good question. Perhaps it is better to instead use the leader key also for this, e.g.

```
`. -> \dots
```

This would follow the same pattern as the other mappings, such as

```
`a -> \alpha
`b -> \beta
```

**Comment by lervag** [comment]
I've begun to think that this feature should not interfere, and so I think what I just proposed might be a good solution.

@Konfekt What do you think?

**Comment by Konfekt** [comment]
This screams for an iabbreviaton: iabb ... ldots. Never encountered an occasion where an imap was necessary. In @rmano s case, the iabb would only be triggered if the first comma is followed by a space.

**Comment by Rmano** [comment]
In effect 
```
 foreach \x in {1,2,     ...,9} 
```
is legal --- you can have a space after the first comma. What is not legal is 
```
 foreach \x in {1,2,... ,9} 
```
---you cannot have a space after the three dots. But nonetheless, I think that @Konfekt may be right; I never type a space there (basically because I can't remember where the space is legal and where is not). 

Moreover, the abbreviation could avoid the annoying "pause and eat the next char temporarily"  every time you insert a `.`  in a line (it happens also with the opening delimiters `([{`; not a big problem though when you get used to it).

**Comment by lervag** [comment]
Yes, I agree, this could really warrant an `iabbr`. In any case, I think it should be removed as a default mapping. I'm doing so now. If anyone complains, I'll suggest that they either readd the map through vimtex or by using a manual iabbrevation. This should also fix you problem.

**Comment by lervag** [comment]
Fixed in e982722e89462716e2ecdb6ce7aa9baea6dc8c98.
